readme.md

```diff
@@ -577,6 +577,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit) - Open source Java app performance management tool.
+- [Parasite](https://github.com/ecohealthalliance/awesome-parasite) - Resources for parasites and host-pathogen interactions.
```

Comment:
> The entry in the Awesome list should:
Include a short description **about the project/theme of the list**. It should not describe the list itself.

⬆️